WebRoot/skins/_base/base4/skin.css

```diff
@@ -782,14 +787,21 @@ svg.Img_ImSmallDnD {
 /*Adding background for svg's with gradient via css background rule
   This can be removed once we have support for gradient's to be rendered with <use>
 */
-
 div.ImgTagStack {
     background: url("../img/zimbra.svg?v=@jsVersion@#ImgTagStack-view") no-repeat 2px center;
     background-size: 100% 100%;
     display: inline-block;
     @SmallIconSize@
 }
 
+/* Applying height/width to loader svg icon. This would not be required once we have
+    support for generating css with height/width from file name.
+*/
+div.ImgLoader-view {
+    height: 18px !important;
```

Comment:
Not sure if it has to be `18px`? Can we use `SmallIconWidth` and `SmallIconHeight` here? Or `SmallIconSize`?